python/ray/tune/util.py

```diff
@@ -1,14 +1,18 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
 
-import logging
 import base64
+import copy
 import fnmatch
+import logging
 import os
-import copy
-import numpy as np
+import threading
 import time
+from collections import defaultdict
+from threading import Thread
+
+import numpy as np
+import psutil
+import GPUtil
```

Comment:
Can you move these imports into `UtilMonitor`? Otherwise this forces both of these to be dependencies of Tune

Reply:
specifically psutil and GPUtil